dev-libs/glib/glib-2.80.4.ebuild

```diff
@@ -0,0 +1,320 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_REQ_USE="xml(+)"
+PYTHON_COMPAT=( python3_{11..13} )
```

Comment:
Please keep 3.10 support to avoid breaking reverse deps:
https://bugs.gentoo.org/936437

Reply:
Done in https://github.com/gentoo/gentoo/pull/36834/commits/78213b281a9371b220b4ec0969107c89f1c770ec